packages/MonorepoBuilder/packages/Split/src/PackageToRepositorySplitter.php

```diff
@@ -89,9 +96,12 @@ public function splitDirectoriesToRepositories(array $splitConfig, string $rootD
         $this->reportFinishedProcesses();
     }
 
-    private function processActiveProcesses(): void
+    private function processActiveProcesses(?int $numberOfProcessesToWaitFor = null): void
```

Comment:
This should be required value

Reply:
I thought about that and wasn't sure which path to take. I will change it.

Reply:
I removed the default value: 5ca36ba6f